modules/button/css/lib/button.scss

```diff
@@ -210,87 +224,59 @@ $_wdc-btn-caret-down-icon-hover: svg-load(
   }
 
   &.wdc-btn-delete {
-    @include wdc-btn-box-shadow($wdc-btn-border-color-delete);
     background-color: $wdc-btn-bg-delete;
     color: $wdc-btn-color-delete;
+    border-color: $wdc-btn-border-delete;
+
+    &:not([disabled]):not(.wdc-btn-disabled) {
+      &.wdc-btn-hover,
+      &:hover {
+        color: $wdc-btn-color-delete-hover;
+        background-color: $wdc-btn-bg-delete-hover;
+        border-color: $wdc-btn-border-delete-hover;
+
+        .wd-icon-accent,
+        .wd-icon-fill {
+          fill: $wdc-btn-color-delete;
+        }
+      }
 
-    &.wdc-btn-hover,
-    &:hover {
-      @include wdc-btn-box-shadow($wdc-btn-border-color-delete-hover);
-      color: $wdc-btn-color-delete-hover;
-      background-color: $wdc-btn-bg-delete-hover;
-    }
-
-    &.wdc-btn-active,
-    &:active {
-      color: $wdc-btn-color-delete-hover;
-      background-color: $wdc-btn-bg-delete-hover;
-    }
+      &.wdc-btn-active,
+      &:active {
+        background-color: $wdc-btn-bg-delete;
+        color: $wdc-btn-color-delete;
+        border-color: $wdc-btn-border-delete;
+      }
 
-    &:focus,
-    &.wdc-btn-focus {
-      color: $wdc-btn-color-delete-hover;
-      background-color: $wdc-btn-bg-delete-hover;
-      box-shadow: 0 0 0 2px $wdc-color-french-vanilla-100, 0 0 0 4px $wdc-btn-border-color-focus;
+      &:focus,
+      &.wdc-btn-focus {
+        background-color: $wdc-btn-bg-delete;
+        color: $wdc-btn-color-delete;
+        box-shadow: 0 0 0 2px $wdc-color-french-vanilla-100, 0 0 0 4px $wdc-btn-border-focus;
+        border-color: $wdc-btn-border-delete-active;
+
+        &.wdc-btn-hover,
+        &:hover {
+          color: $wdc-btn-color-delete-hover;
+          background-color: $wdc-btn-bg-delete-hover;
+          border-color: $wdc-btn-border-delete-hover;
+
+          &.wdc-btn-active,
+          &:active {
+            background-color: $wdc-btn-bg-delete;
+            color: $wdc-btn-color-delete;
+            border-color: $wdc-btn-border-delete-active;
+          }
+        }
+      }
     }
 
     &.wdc-btn-disabled,
     &:disabled {
-      @include wdc-btn-box-shadow($wdc-btn-border-color-delete-disabled);
+      box-shadow: none;
       color: $wdc-btn-color-delete-disabled;
       background-color: $wdc-btn-bg-delete-disabled;
-    }
-  }
-
-  &.wdc-btn-icon {
```

Comment:
Last change - I promise haha. Can we add this back in and make it match the default in react? (i.e. `wdc-btn-icon` inherits all the styles from `wdc-btn-icon-circle`). 

Can you also include the `.wdc-btn-medium` size by default?

Can you put these changes in the icon button file instead of this one?

This will require much less markup for the most common use cases.